harvester_e2e_tests/integrations/test_1_volumes.py

```diff
@@ -88,6 +149,11 @@ def test_create_volume(api_client, unique_name, ubuntu_image, create_as, source_
     polling_for("volume do created",
                 lambda code, data: 200 == code and data['status']['phase'] == "Bound",
                 api_client.volumes.get, unique_name)
+    code2, data2 = api_client.images.get(ubuntu_image['display_name'])
+    # This grabs the failed count for the image
+    failed: int = data2['status']['failed']
+    # This makes sure that the failures are 0
+    assert 0 == failed, 'Image failed more than 3 times'
```

Comment:
Is it better to use `assert 3 >= failed, 'Image should NOT failed more than 3 times'`?

Reply:
Fixed.

Reply:
You're right, that's not a clear message. I changed it to `Image download correctly failed more than 3 times with bad checksum`